test/functional/feature_xfield_maxblocksize.py

```diff
@@ -32,7 +32,7 @@
 from test_framework.script import CScript, OP_CHECKSIG, OP_TRUE, SignatureHash, SIGHASH_ALL
 from test_framework.messages import CTransaction, MAX_BLOCK_BASE_SIZE, CTxOut, CTxIn, COutPoint, uint256_from_str, ser_compact_size, msg_headers, CBlockHeader
 
-MAX_BLOCK_SIGOPS = 20000
+MAX_BLOCK_SIGOPS = int((100000 - 1000) / 50)
```

Comment:
What is the basis for this formula?

Reply:
20000 would be the value of  MAX_BLOCK_SIZE/50. we now need it to be DEFAULT_BLOCK_MAX_SIZE/50 as DEFAULT_BLOCK_MAX_SIZE is the initial value in XFieldHistory for MaxBlockSize